packages/@sanity/form-builder/src/inputs/PortableText/text/TextBlock.styles.ts

```diff
@@ -0,0 +1,161 @@
+
+interface TextBlockStyleProps {
+  $level?: number
+}
+
+function textBlockStyle(props: TextBlockStyleProps & {theme: Theme}) {
+  const {$level, theme} = props
+  const {color, fonts, radius, space} = theme.sanity
+  const numberMarker = TEXT_NUMBER_FORMATS[($level - 1) % TEXT_NUMBER_FORMATS.length]
+  const bulletMarker = TEXT_BULLET_MARKERS[($level - 1) % TEXT_BULLET_MARKERS.length]
+
+  return css`
+    --marker-bg-color: transparent;
+
+    mix-blend-mode: ${color.dark ? 'screen' : 'multiply'};
+    position: relative;
+
+    & > [data-ui='TextBlock_inner'] {
+      position: relative;
+      flex: 1;
+    }
+
+    & > div:before {
+      content: '';
+      position: absolute;
+      top: -${space[1]}px;
+      bottom: -${space[1]}px;
+      left: -${space[1]}px;
+      right: -${space[1]}px;
+      border-radius: ${radius[2]}px;
+      background-color: var(--marker-bg-color);
+    }
+
+    &[data-custom-markers] {
+      --marker-bg-color: ${color.dark ? hues.purple[950].hex : hues.purple[50].hex};
+    }
+
+    &[data-warning] {
+      --card-border-color: ${color.muted.caution.enabled.border};
+      --marker-bg-color: ${color.muted.caution.hovered.bg};
+    }
+
+    &[data-error] {
+      --card-border-color: ${color.muted.critical.enabled.border};
+      --marker-bg-color: ${color.muted.critical.hovered.bg};
+    }
+
+    & [data-list-prefix] {
+      position: absolute;
+      margin-left: -4.5rem;
+      width: 3.75rem;
+      text-align: right;
+      box-sizing: border-box;
+    }
+
+    &[data-list-item='number'] [data-list-prefix] {
+      font-variant-numeric: tabular-nums;
+
+      & > span:before {
+        content: counter(${createListName($level)}) '.';
+        content: counter(${createListName($level)}, ${numberMarker}) '.';
+      }
+    }
+
+    &[data-list-item='bullet'] [data-list-prefix] {
+      & > span {
+        position: relative;
+        top: -0.1875em;
+
+        &:before {
+          content: '${bulletMarker}';
+          font-size: 0.46666em;
+        }
+      }
+    }
+
+    & [data-text] {
+      overflow-wrap: anywhere;
+      text-transform: none;
+      white-space: pre-wrap;
+      font-family: ${fonts.text.family};
+      flex: 1;
+
+      *::selection {
+        background-color: ${color.selectable.primary.pressed.bg};
+      }
+    }
+  `
+}
+
+export const TextRoot = styled.div<TextBlockStyleProps>(textBlockStyle)
+
+export const TextBlockFlexWrapper = styled(Flex)`
+  position: relative;
+`
+
+export const ListPrefixWrapper = styled.div`
+  user-select: none;
+  white-space: nowrap;
+`
+
+export const BlockActionsOuter = styled(Box)`
+  line-height: 0;
+  width: 25px;
+  position: relative;
+  user-select: none;
+
+  /* Without this, select all (CMD-A) will not work properly */
+  /* when the editor is in non-fullscreen mode. */
+  &:before {
+    content: ' ';
+    font-size: 0;
+  }
+`
+
+export const BlockActionsInner = styled(Flex)`
+  position: absolute;
+  right: 0;
+  top: -7px;
```

Comment:
Should this value be calculated with theme values?

Reply:
True, I’ve added a commit to fix this 👍